src/nft-card-back.ts

```diff
@@ -440,10 +475,10 @@ export class NftCardBackTemplate extends LitElement {
    * @param trait - The object containing an asset's trait
    * @return true if the trait is a 'Property' and false otherwise
    */
-  private isProperty(trait: object, collectionTraits: object) {
+  private isProperty(trait: Trait, collectionTraits: CollectionTraits) {
     return (
       trait.display_type === null &&
-      !('max' in collectionTraits[trait.trait_type])
+      trait.trait_type in collectionTraits ? !('max' in collectionTraits[trait.trait_type]) : true
```

Comment:
this logic looks like it could be simplified to get rid of returning a literal bool